packages/DataGrid/stories/DataGrid.rowindicator.stories.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import * as Sbook from "storybook/assets/styles/common.styles";
+import DataGrid from "../src";
+import fixtures from "./helpers/fixtures";
+
+const data = fixtures(10);
+// <DataGrid.ColumnDefinition
+//   width={40}
+//   headerA11yText={() => "unchecked"}
+//   cellA11yText={() => "unchecked"}
+//   header={props => (
+//     <DataGrid.RowIndicator hasIndexIndicator={false} isChecked={() => "unchecked"} {...props} />
+//   )}
+//   cell={props => <DataGrid.RowIndicator isChecked={() => "unchecked"} {...props} />}
+// />
+
+export function App() {
```

Comment:
Is this story still functional and showing the row indicator component? I see its commented out at the top

Reply:
it's not but I'm thinking of deprecating this component until we have the need to use it, I think right now we can do what you did with the checkbox on the mission control what do you think?

Reply:
Let me talk to you a bit about it, ill pm u